.github/workflows/build.yml

```diff
@@ -50,6 +50,40 @@ jobs:
         name: test-results
         if-no-files-found: error
         path: '**/target/*-reports/*.xml'
+  build-launcher-win64:
+    runs-on: windows-2019
+    env:
+      PROGRAM_OUTPUT: eclipse.exe
+      PROGRAM_LIBRARY: eclipse.dll
+      DEFAULT_OS: win32
+      DEFAULT_OS_ARCH: x86_64
+      DEFAULT_WS: win32
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 1
+    - name: Set up JDK 8
+      uses: actions/setup-java@v3
+      with:
+        java-version: '8'
+        distribution: 'temurin'
+    - name: Visual Studio shell
+      uses: egor-tensin/vs-shell@v2
```

Comment:
Would the [microsoft/setup-msbuild](https://github.com/microsoft/setup-msbuild) action work as well?
Since it is provided by Microsoft directly I would expect/hope that it is better supported.

Reply:
This was my first github action and I googled for something that defined Visual Studio 2019 (the version referred to in the build.bat file) and it was one of the first matches.
Feel free to try out if the other action works and if so, change it.

My sole goal with this setup was to make it possible to validate #118 without installing the required tools (including buying them if needed).

Reply:
No problem :)
Then lets see what happens in https://github.com/eclipse-equinox/equinox/pull/250.